Missionframework/modules/11_permission/fnc/fn_permission_syncClients.sqf

```diff
@@ -0,0 +1,49 @@
+/*
+    KPLIB_fnc_permission_syncClients
+
+    File: fn_permission_syncClients.sqf
+    Author: KP Liberation Dev Team - https://github.com/KillahPotatoes
+    Date: 2018-12-22
+    Last Update: 2018-12-25
+    License: GNU General Public License v3.0 - https://www.gnu.org/licenses/gpl-3.0.html
+
+    Description:
+        Receive client variables and send them to all clients.
+
+    Parameter(s):
+        _list       - KPLIB_permission_list      [ARRAY, defaults to []]
+        _default    - KPLIB_permission_default   [ARRAY, defaults to []]
+        _types      - KPLIB_permission_types     [ARRAY, defaults to []]
+        _groups     - KPLIB_permission_groups    [ARRAY, defaults to []]
+
+    Returns:
+        Function reached the end [BOOL]
+*/
+
+params [
+    ["_list", [], [[]]],
+    ["_default", [], [[]]],
+    ["_types", [], [[]]],
+    ["_groups", [], [[]]]
+];
+
```

Comment:
If I understand it correctly this should only run on the server?
Therefore it would needed to be `remoteExecCall`ed by the clients and here could be a `if (!isServer) exitWith {};`